requirements.txt

```diff
@@ -1,5 +1,4 @@
 
-Jinja2==2.11.2
 Markdown==3.3.4
 mkdocs>=1.1.2
 mkdocs-versioning==0.4.0
```

Comment:
You might not need version tags on anything in here. I removed all those from PMM as Netilfy and Github builds work ok with the latest of everything.

Reply:
Fixed